src/ir/operator_test.cc

```diff
@@ -0,0 +1,36 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------------
+#include "src/ir/operator.h"
+
+#include "src/common/testing/gtest.h"
+
+namespace raksha::ir {
+namespace {
+
+class OperatorTest : public testing::TestWithParam<absl::string_view> {};
+
+TEST_P(OperatorTest, NameIsCorrectlySet) {
+  const auto& name = GetParam();
```

Comment:
Since the parameter is a `string_view`, which is naively copyable, shouldn't this just be `absl::string_view` instead of this `auto` ref (which I belive will become `const absl::string_view &name`)?

Reply:
Yes, will do. Was an autopilot. :)